pom.xml

```diff
@@ -484,7 +484,7 @@
 
         <andes.client.version>3.3.12</andes.client.version>
         <azure.messaging.eventhubs.version>5.6.0</azure.messaging.eventhubs.version>
-        <carbon.apimgt.version>9.18.12</carbon.apimgt.version>
+        <carbon.apimgt.version>9.18.2</carbon.apimgt.version>
```

Comment:
Is it fine to downgrade carbon-apimgt version here ? We updated the above version with M4.